### PR TITLE
refac(console): match span id earlier

### PR DESCRIPTION
**Pull request by 0xPoe**
Follow the same pattern applied in #533 to match the span id
earlier to avoid checking twice.

### CODE REVIEW

File: tokio-console/src/view/mod.rs [hunk 1]
```diff
@@ -69,6 +69,7 @@ pub(crate) enum UpdateKind {
     SelectTask(u64),
     /// The TaskView is exited
     ExitTaskView,
+    #[allow(dead_code)]
```
Comment (0xPoe) [inline tokio-console/src/view/mod.rs]: We might use this in the future. So I just allow the dead code check. Please let me know if we want to remove it.

**Review by 0xPoe — COMMENTED** [review]
🔢 Self-check (PR reviewed by myself and ready for feedback.)

File: tokio-console/src/view/mod.rs (hunk 1 above)
Comment (hds) [reply tokio-console/src/view/mod.rs]: I'm not sure why this isn't used but `SelectTask(u64)` is.

**Review by hds — CHANGES_REQUESTED** [review]
This looks good.

Would you mind splitting the change to make clippy happy into another PR though? It's best to not mix concerns.

File: tokio-console/src/view/mod.rs (hunk 1 above)
Comment (0xPoe) [reply tokio-console/src/view/mod.rs]: I think we used the task ID to monitor task details, but we don't need to use it for resources.

**Comment by 0xPoe** [comment]
> Would you mind splitting the change to make clippy happy into another PR though? It's best to not mix concerns.

Dropped. I will open a new PR for it.

**Comment by 0xPoe** [comment]
> > Would you mind splitting the change to make clippy happy into another PR though? It's best to not mix concerns.
> 
> Dropped. I will open a new PR for it.

See: https://github.com/tokio-rs/console/pull/536

**Review by hds — APPROVED** [review]
This looks good. Thank you very much!

Could you please make sure that the commit message follows our styling for the `console` repo. This is the basis for our release notes, so it's important. We follow [conventional commits], but checking the summary in the **Commit message guidelines** section of the [CONTRIBUTING.md] document is probably enough.

In this case, the commit message should be something like (both `refac` and `refactor` are accepted, see https://github.com/tokio-rs/console/blob/main/cliff.toml#L80):

```
refac(console): match span id earlier (#535)

Follow the same pattern applied in #533 to match the span id
earlier to avoid checking twice.
```

[conventional commits]: https://www.conventionalcommits.org/en/v1.0.0/
[CONTRIBUTING.md]: https://github.com/tokio-rs/console/blob/main/CONTRIBUTING.md#commit-message-guidelines

**Comment by 0xPoe** [comment]
@hds Updated. Thanks! I didn't know that we also needed to include a detailed commit body. I thought that the Pull Request body was enough, although my PR body wasn't very good either. I'll make sure to add it next time.

**Pull request merged**